lib/lndclient/LndClient.ts

```diff
@@ -949,8 +958,12 @@ class LndClient extends SwapClient {
       return;
     }
 
-    const deleteChannelBackupSubscription = () => {
+    const deleteChannelBackupSubscription = async () => {
       this.channelBackupSubscription = undefined;
+      // setting status as disconnected because it's reasonable to assume
+      // that the swap client ended the subscription because it's disconnected
```

Comment:
I'm thinking it might be best to scrap this method and in its place use the `disconnect` method we already have. Then we'd also add a check there to close `channelBackupSubscription` and set it to undefined (unless it's already undefined). It's a bit counterintuitive to me that `deleteChannelBackupSubscription` also disconnects the client, even tho this comment does help to clear it up.